bowtie/_smoke.py

```diff
@@ -215,6 +216,40 @@ def __rich_console__(
                 padding=2,
             )
             epilog.add_row(panel)
+
+            test = {
```

Comment:
What's making you create a new test here? We want to show one of the ones that just failed, and tell the user how to run something which does that.